spec/integration/search/search_spec.rb

```diff
@@ -216,6 +216,34 @@
     expect(aggregates_without_filter["mainstream_browse_pages"]["options"].size).to eq(2)
   end
 
+  it "aggregate_counting_with_filter_on_a_different_field" do
```

Comment:
Rspec test name don't require the underscore. 

I have purposefully left the underscore in the migrated tests as a marker they were migrated, with the plan to remove them as they are reviewed/updated. 🍰 

should maybe have mentioned that somewhere.....

Reply:
We discussed this but there are so many tests with underscores in the name, it would look inconsistent.

Reply:
We have now decided to move the aggregates tests into their own file, removing the underscores in the process for those tests.